### PR TITLE
logs errors when parsing public keys

### PR DESCRIPTION
**Pull request by favadi**
It is impossible to debug why public key doesn't work because we are ignoring the returned errors.

### CODE REVIEW

File: easyssh.go [hunk 1]
```diff
@@ -80,13 +81,17 @@ func getSSHConfig(config DefaultConfig) *ssh.ClientConfig {
 	}
 
 	if config.KeyPath != "" {
-		if pubkey, err := getKeyFile(config.KeyPath); err == nil {
+		if pubkey, err := getKeyFile(config.KeyPath); err != nil {
+			log.Printf("getKeyFile: %v", err)
```
Comment (appleboy) [inline easyssh.go]: missing `\n`

File: easyssh.go [hunk 2]
```diff
@@ -80,13 +81,17 @@ func getSSHConfig(config DefaultConfig) *ssh.ClientConfig {
...
+		} else {
 			auths = append(auths, ssh.PublicKeys(pubkey))
 		}
 	}
 
 	if config.Key != "" {
-		if signer, err := ssh.ParsePrivateKey([]byte(config.Key)); err == nil {
+		if signer, err := ssh.ParsePrivateKey([]byte(config.Key)); err != nil {
+			log.Printf("ssh.ParsePrivateKey: %v", err)
```
Comment (appleboy) [inline easyssh.go]: missing `\n`

Comment (favadi) [inline easyssh.go]: fixed in d2b90f9.

**Comment by appleboy** [comment]
@favadi Thanks.

**Pull request merged**